gefserver/pier/pier.go

```diff
@@ -146,6 +150,42 @@ func (p *Pier) BuildService(buildDir string) (db.Service, error) {
 	return service, nil
 }
 
+// startTimeOutTicker starts a clock that checks if a job exceeds an execution timeout
+func (p *Pier) startTimeOutTicker(jobId db.JobID) {
+	ticker := time.NewTicker(time.Second * time.Duration(p.checkInterval))
+	go func() {
+		for range ticker.C {
+			job, err := p.db.GetJob(jobId)
+
+			if err != nil {
+				p.db.SetJobState(job.ID, db.NewJobStateError("Cannot get information about the job running", 1))
+				ticker.Stop()
+				break
+			}
+			if job.State.Code != -1 {
+				ticker.Stop()
+				break
+			}
+
+			startingTime := job.Created
+			currentTime := time.Now().UTC()
```

Comment:
this one is UTC, but all the ones we create in the db are local times; we should make an issue to transform them all